minecraft_servers/opinsel/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+    "server_name": "opinsel",
+    "nice_name": "OpInsel",
+    "direct_ip": "mc.opinsel.de",
+    "server_wildcards": [
+      "%.opinsel.de"
+    ],
+    "supported_languages": [
+      "de"
+    ],
+    "social": {
+      "web": "https://opinsel.de",
+      "web_shop": "https://opinsel.de/shop",
+      "discord": "https://opinsel.de/discord"

```

Comment:
Custom invites reserved for partners; use https://discord.gg/[INVITE]

Reply:
https://discord.com/invite/Mn9drhfaDQ